trikot-viewmodels-declarative-flow/viewmodels-declarative-flow/src/commonMain/kotlin/com/mirego/trikot/viewmodels/declarative/controller/VMDViewModelController.kt

```diff
@@ -3,25 +3,17 @@ package com.mirego.trikot.viewmodels.declarative.controller
 import com.mirego.trikot.foundation.concurrent.atomic
 import com.mirego.trikot.foundation.ref.weakAtomicReference
 import com.mirego.trikot.viewmodels.declarative.viewmodel.VMDViewModel
-import kotlinx.coroutines.CoroutineExceptionHandler
 import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.SupervisorJob
-import kotlinx.coroutines.cancel
 
 abstract class VMDViewModelController<VM : VMDViewModel, N : VMDNavigationDelegate> :
     VMDPlatformViewModelController() {
 
     var navigationDelegate: N? by weakAtomicReference()
 
-    open val exceptionHandler: CoroutineExceptionHandler = CoroutineExceptionHandler { _, throwable ->
-        println("CoroutineExceptionHandler got $throwable")
-    }
-
-    protected val viewModelControllerScope = CoroutineScope(Dispatchers.Main.immediate + SupervisorJob() + exceptionHandler)
-
     abstract val viewModel: VM
 
+    val viewModelScope: CoroutineScope = platformViewModelScope
```

Comment:
Do we really need this val? Or we could only have platformViewModelScope renamed to viewModelScope?

Reply:
Oddly, if I didn't put it there the compiler is unable to see the expected underlying val.